### PR TITLE
New Section: `kit build-start-package`

### PR DESCRIPTION
**Pull request by jurij-jukic**
@nick1udwig, please proof read the section.
most of it was copy pasted from `kit build` and `kit start-package`

things to check:
 - `--no-ui` explanation
 - initial text and discussion

Q: what should be written under `--features`?

### CODE REVIEW

File: src/kit/build-start-package.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+or
+
+```
+kit build-start-package
+```
+
+## Discussion
+
+`kit build-start-package` runs `kit build` followed by `kit start-package`.
```
Comment (nick1udwig) [inline src/kit/build-start-package.md:17]: SHould we link here, e.g. something like
"""
`kit build-start-package` runs [`kit build`](./build.md) followed by `kit start-package`(./start-package.md).
"""
?
(check the links if you copy that suggestion -- they were written by hand and may be wrong)

**Review by nick1udwig — APPROVED** [review]
One nit, otherwise looks great! Thanks for calling out what to look at in particular.

> Q: what should be written under --features?

Something like:
```
Build the package with the given [cargo features](https://doc.rust-lang.org/cargo/reference/features.html).
```

Once you've addressed nit & added to `--features`, ready to merge :rocket:

File: src/kit/build-start-package.md (hunk 1 above)
Comment (jurij-jukic) [reply src/kit/build-start-package.md:17]: will do, good suggestion

**Comment by nick1udwig** [comment]
> @nick1udwig could you give an example of using `--features` with multiple features? i assume cargo.toml needs to be modified as well.

https://doc.rust-lang.org/cargo/reference/features.html#command-line-feature-options

Currently the only feature we support system-wide is `simulation-mode`, but that doesn't stop users from making their own for their processes

**Review by dr-frmr — APPROVED** [review]

**Comment by nick1udwig** [comment]
Oh, one request before merge. Please add a link to `kit-dev-toolkit.md` as well.

**Comment by jurij-jukic** [comment]
@nick1udwig please confirm my `--features` description, then it's good to go

**Review by nick1udwig — APPROVED** [review]
Good :+1: 

Sorry, thought I had approved this one

**Pull request merged**